DESCRIPTION

```diff
@@ -17,5 +17,6 @@ LinkingTo: Rcpp
 Suggests: tinytest
 URL: https://github.com/eddelbuettel/rcppfastfloat/
 BugReports: https://github.com/eddelbuettel/rcppfastfloat/issues
-RoxygenNote: 6.0.1
+RoxygenNote: 7.1.1
+Roxygen: list(markdown = TRUE)
```

Comment:
Don't change it now, but I have a slight preference for the old markdown 6.0.1 which I have in an opt-in directory.  It does not do the stooopid dance of forcing a recompilation and rebuild only to deal with Rd creation.  And with that I often say no to markdown too.  (At least in stull-small projects such as this. Exceptions can be made and are made.)

Reply:
I also realize that it is awkward with collaborators who won't have 6.0.1 around. Ah well. RStudio lacks flexibility in invoking alternate converters, sadly.

Reply:
No worries, just help remind me before if I forget/don't get to it before you're looking to ship to CRAN.

Reply:
We can probably keep it. I am on the fence on md-in-Rd too.  It's on in some projects, I am just not too consisten.  No worries right now.  This all looks _svelte_ as usual.